kANe/v3.md

```diff
@@ -1454,6 +1474,8 @@ Society'. read 'naugamanabhave'..
 
 last line 
 
+## 01 Rājadharma - Introduction
```

Comment:
Thanks for pointing it out. One way of dealing with it is as you have done. Alternatively, one can create subdivisions like:

```
## 0 
## 1 rAjadharma
### 1 Introduction
...
## 2 vyavahAra
### 11 Introduction
...
### 12 Bhukti
...
## 3 sadAchAra
....
## 4 Tail matter (Instead of ## 36 )
...
```

It might be better to go with this structure then.

Reply:
So, sir, shall I create the subdivisions as you have shown and send another PR?

Reply:
Yes, please.

Reply:
okay, sir.

Reply:
Done, sir. #23